lib/resources/http.rb

```diff
@@ -0,0 +1,49 @@
+# encoding: utf-8
+# copyright: 2017, Criteo
+# author: Guilhem Lettron
+# license: All rights reserved
```

Comment:
We do not allow `All rights reserved`. Every contribution needs to be Apache 2 licensed